doc/use_apps/python_tutorial.rst

```diff
@@ -6,7 +6,7 @@ Install
 
 The CCF Python tools package can be used to interact with an existing running service and provides utilities to:
 
-- Issue HTTP requests over TLS to CCF applications
+- Issue HTTP requests over TLS to CCF
```

Comment:
Well it is to CCF application or CCF services, technically. CCF is just the framework, it is not a running instance of a service.

Reply:
```suggestion
- Issue HTTP requests over TLS to a CCF application
```

Reply:
CCF application is too specific IMO - governance and operator requests can also be issued with the Python client. I'd rather go with "a CCF service".